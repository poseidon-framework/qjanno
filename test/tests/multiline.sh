qhs -H -O "SELECT foo,bar,quux FROM multiline.csv LIMIT 1"
