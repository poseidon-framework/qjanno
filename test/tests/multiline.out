foo bar quux
a0 1 c2
